pkg/geo/geographiclib/geographiclib.go

```diff
@@ -109,3 +114,24 @@ func (s *Spheroid) AreaAndPerimeter(points []s2.Point) (area float64, perimeter
 	)
 	return float64(areaDouble), float64(perimeterDouble)
 }
+
+// Project returns computes the location of the projected point.
+//
+// Using the direct geodesic problem from GeographicLib (Karney 2013).
+func (s *Spheroid) Project(point s2.LatLng, distance float64, azimuth float64) s2.LatLng {
+	var lat, lng C.double
+
+	C.geod_direct(
+		&s.cRepr,
+		C.double(math.Pi*point.Lat/180.0),
+		C.double(math.Pi*point.Lng/180.0),
+		C.double(azimuth*180.0/math.Pi),
+		C.double(distance),
+		&lat,
+		&lng,
+		nil)
+
+	return s2.LatLng{
+		Lat: s1.Angle(float64(lat) * math.Pi / 180.0),
```

Comment:
s1.Angle(lat * s1.Degree) may be good here.

Reply:
I fixed this with s2.LatLngFromDegrees.